src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -36,12 +36,31 @@
 
     private static final long serialVersionUID = 1;
 
-    public PullRequestSCMHead(int number) {
+    private String user;
```

Comment:
> in the PR context, the right concept is `assignee`

No, that is something else: the person the PR is “assigned to”, which in most PRs is not set at all.

We are looking at `user.login` of the PR API response, as reflected in the Java call

```java
ghPullRequest.getUser().getLogin()
```

So I still maintain that `user` is the natural field name here.